### ISSUE TITLE
Custom Orders

### ISSUE THREAD

**Issue by npruehs** [npruehs/ue4-rts#122]
Allow users to create new custom orders with behavior trees.

_This is a breaking change and thus requires an increase of the major version._

**Comment by JernejHabjan** [comment]
How would attack - move order be implemented? Does this belong under custom orders?

**Comment by npruehs** [comment]
Yes, definitely. With A Year Of Rain, we found that attack-move is an incredibly complex order (more than meets the yet). Most notably, decisions like when to return from an acquired enemy (and where to) are not easily made and implemented, and we had various bugs with units ending up in locations where they shouldn't (especially when repeatedly disengaging and encountering enemies on the way).
